src/index.js

```diff
@@ -24,15 +25,37 @@ const displayTasks = () => {
   allTasks.innerHTML = '';
   for (let i = 0; i < taskList.length; i += 1) {
     const each = taskList[i];
-    const list = `<div class="eachTask">
-      <div class="group-list">
-      <input type="checkbox" class="box" id="list-box" name="list-box">
-        <p class="task-name">${each.description}</p>
-      </div> 
-      <button class="menu-icon" id="${each.id}"><i class="fas fa-ellipsis-v"></i></button>
-    </div>
-    <hr>`;
-    allTasks.innerHTML += list;
+
+    const eachTask = document.createElement('div');
+    eachTask.className = 'eachTask';
+
+    const list = document.createElement('div');
+    list.className = 'group-list';
+
+    const input = document.createElement('input');
+    input.setAttribute('type', 'checkbox');
+    input.setAttribute('class', 'check-box');
+    input.id = each.id;
+    input.checked = each.completed;
+    input.addEventListener('change', () => {
+      completedStatus(each, taskList);
+    });
```

Comment:
- [ ] After refresh, The `UI` still not showing the updated status of the checkbox, but I can see the local storage is updated. Please fix the UI.

![Screenshot from 2021-08-25 19-55-57](https://user-images.githubusercontent.com/22434660/130810466-752037a5-185f-46be-b61b-042757a38808.png)